### ISSUE TITLE
qgis-26 install fails on OSX 10.7.5

### ISSUE THREAD

**Issue by sharkinsspatial** [OSGeo/homebrew-osgeo4mac#82]
As referenced in #63, #48, #70, #52, #45 I'm unable to install qgis-26.  I've tried all the suggestions in these issues but when running
````
brew update
brew reinstall expat
brew reinstall qgis-26
````
I still receive
````
dyld: Library not loaded: /usr/local/lib/libexpat.1.dylib
  Referenced from: /usr/local/opt/cmake/bin/cmake
  Reason: Incompatible library version: cmake requires version 8.0.0 or later, but libexpat.1.dylib provides version 7.0.0
````

**Comment by dakcarto** [comment]
Hi @sharkinsspatial,

Thanks for reporting. The failed dylib loading is:

```
Referenced from: /usr/local/opt/cmake/bin/cmake
```
Which means you need to try:

```sh
brew reinstall cmake
```

**Comment by sharkinsspatial** [comment]
Upgraded cmake and now failing on on the gdal_dynamic build
````
==> ./configure --prefix=/usr/local/Cellar/grass-64/6.4.4_1 --enable-largefile --enable-shared --with-c
==> make GDAL_DYNAMIC=
appear in the error log. If you get an error building a library, you will
also get errors from anything which uses the library.
````

**Comment by dakcarto** [comment]
@sharkinsspatial If it is failing to install `grass-64`, as part of the `qgis-26` formula install, you need to try installing `grass-64` separately and provide the [gist logs for its failed build, as indicated in Homebrew's Troubleshooting](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Troubleshooting.md). Since you are more than likely compiling across multiple CPU cores, whatever the last thing listed is, is probably not the error, which will be in the full logs.

**Comment by sharkinsspatial** [comment]
https://gist.github.com/anonymous/80c27a33d5ba15eb2dd8

**Comment by dakcarto** [comment]
It looks like you have a couple issues that need ironed-out first:

* Fix everything you can noted in `doctor.out`, especially upgrading you Xcode, since the first fatal error is: `fatal error: error in backend: ran out of registers during register allocation`. That appears to be an issue with clang, which should be upgraded when moving from Xcode 4.5 to Xcode 4.6.3.

* Ensure you fix the Python path issue noted in `doctor.out`. This is very important to `wxpython` which needs to read `.pth` files. GRASS uses `wxpython`.

**Comment by sharkinsspatial** [comment]
I'm on OSX 10.7.5 (no Xcode installed, only CLT from April 2013, the last available for Lion), so I don't believe I'm able to update my CLT to the version with Xcode 4.6.3.

**Comment by dakcarto** [comment]
The last available Xcode for Lion is 4.6.3, with 4.6.2 Command Line Tools. You can get the CLT from [Apple's developer site](http://adcdownload.apple.com/Developer_Tools/command_line_tools_os_x_mountain_lion_for_xcode__april_2013/xcode462_cltools_10_86938259a.dmg). If not, i.e. you need a developer account, I can post the CLT to my qgis.dakotacarto.com site for you (~146 MB).

**Comment by sharkinsspatial** [comment]
Upgraded to the last available CLT and success, thanks so much for your assistance, sorry for the difficulties.
